neevaco/neeva-ios
# Immediately add new Space tabs to tab group

Immediately adds new Space tabs to a tab group. It used to be the case that the app would need to be restarted for the change to appear.

* More details about the changes
* Even more details about the changes
* Yet more details about the changes

## Checklists

### How was this tested?

### Manual test cases
Open one item from a Space, open a second item from the same Space. Verify that they are both in a tab group.

### Screenshots and screen recordings

https://user-images.githubusercontent.com/41495718/162075524-12c9a5fc-c1f6-4db5-bfb4-f3437f69bbe2.mp4



## Issues addressed
* Fixes #3257
